runtime/meta/ffi.cpp

```diff
@@ -103,8 +106,48 @@ extern "C" {
     } else if (symbol == tag_type_longdouble()) {
       return &ffi_type_longdouble;
     } else {
-      throw std::invalid_argument("Arg is not a supported type");
+      return NULL;
```

Comment:
let's add a case here for structs rather than exposing the logic relating to structs outside of getTypeFromSymbol.